pkgs/applications/editors/emacs/elisp-packages/manual-packages/eglot-booster/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  melpaBuild,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  emacs-lsp-booster,
+}:
+
+melpaBuild {
+  pname = "eglot-booster";
+  version = "0-unstable-2025-07-16";
+
+  src = fetchFromGitHub {
+    owner = "jdtsmith";
+    repo = "eglot-booster";
+    rev = "cab7803c4f0adc7fff9da6680f90110674bb7a22";
+    hash = "sha256-xUBQrQpw+JZxcqT1fy/8C2tjKwa7sLFHXamBm45Fa4Y=";
+  };
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  postPatch = ''
+    substituteInPlace eglot-booster.el \
+      --replace "executable-find \"emacs-lsp-booster\"" \
+                "executable-find \"${lib.getExe emacs-lsp-booster}\""
+  '';
```

Comment:
There are many string literals `"emacs-lsp-booster"` in `eglot-booster`.  I am not sure if it is enough to change only one string literal.  Maybe it is better to do it in a future PR so that we can get this PR merged soon.